cookbook/sm-s0301.rst

```diff
@@ -0,0 +1,216 @@
+                  - switch.is_on: relay1
+                  - switch.is_on: relay2
+                  - switch.is_on: relay3
+                  - switch.is_on: relay4
+                  - switch.is_on: relay5
+              then:
+                - light.turn_on: red_leds
+              else:
+                - light.turn_off: red_leds
+
+      - platform: gpio
+        id: relay2
+        name: ${friendly_name} Relay 2
+        pin:
+        number: GPIO13
+        inverted: true
+        on_turn_on:
+          - light.turn_on: red_leds
+        on_turn_off:
+          - if:
+              condition:
+                or:
+                  - switch.is_on: relay1
+                  - switch.is_on: relay2
+                  - switch.is_on: relay3
+                  - switch.is_on: relay4
+                  - switch.is_on: relay5
+              then:
+                - light.turn_on: red_leds
+              else:
+                - light.turn_off: red_leds
+
+    - platform: gpio
+        id: relay3
+        name: ${friendly_name} Relay 3
+        pin:
+          number: GPIO12
+          inverted: true
+        on_turn_on:
+          - light.turn_on: red_leds
+        on_turn_off:
+          - if:
+              condition:
+                or:
+                  - switch.is_on: relay1
+                  - switch.is_on: relay2
+                  - switch.is_on: relay3
+                  - switch.is_on: relay4
+                  - switch.is_on: relay5
+              then:
+                - light.turn_on: red_leds
+              else:
+                - light.turn_off: red_leds
+
+    # The outlet farthest from the power cord
+      - platform: gpio
+        id: relay4
+        name: ${friendly_name} Relay 4
+        pin:
+          number: GPIO14
+          inverted: true
+        on_turn_on:
+         - light.turn_on: red_leds
+        on_turn_off:
+          - if:
+              condition:
+                or:
+                  - switch.is_on: relay1
+                  - switch.is_on: relay2
+                  - switch.is_on: relay3
+                  - switch.is_on: relay4
+                  - switch.is_on: relay5
+              then:
+                - light.turn_on: red_leds
+              else:
+                - light.turn_off: red_leds
+
+      # The USB plugs
+      - platform: gpio
+        id: relay5
+        name: ${friendly_name} Relay 5
+        pin:
+          number: GPIO16
+        on_turn_on:
+          - light.turn_on: red_leds
+        on_turn_off:
+          - if:
+              condition:
+                or:
+                  - switch.is_on: relay1
+                  - switch.is_on: relay2
+                  - switch.is_on: relay3
+                  - switch.is_on: relay4
+                  - switch.is_on: relay5
+              then:
+                - light.turn_on: red_leds
+              else:
+                - light.turn_off: red_leds
+
+.. _Blakaddr: https://blakadder.github.io/templates/sm-s0301-us.html
```

Comment:
Please follow the standard page layout:

 - Add a See Also section
 - Add seo tags with `.. seo`

Also, you must link this page on the main page, otherwise it does not show up anywhere and is essentially dead code.